src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalTests.cs

```diff
@@ -3603,6 +3603,19 @@ public void TestMaxKeywordLength()
             Assert.Equal(LexerCache.MaxKeywordLength, max);
         }
 
+        [Fact]
+        public void TestNoKind8441InKeywordKinds()
```

Comment:
I didn't find any specific test file for `SyntaxFacts` class, so I put it here (e.g. test above uses `SyntaxFacts`)

Reply:
Actually, found out that there is `SyntaxTests` for that. Moving the test